### PR TITLE
Build: Fix selectors used in Cypress tests and E2E exit code

### PR DESCRIPTION
**Pull request by gaetanmaisse**
## What I did

 - Selectors used in Cypress tests were not pointing to existing element so I replaced them with working ones
 - After a refactoring, E2E tests didn't use the result code of Cypress tests anymore so even if Cypress tests were failing the e2e step was ok
 - In some cases (covered by the e2e extended test suite) a prompt was displayed, causing the CI to fails after a 10min timeout.
 - Reduce CircleCI `no_output_timeout` to 5m for E2E steps
 - Add `@storybook/react-docgen-typescript-plugin` to the list of `@storybook/*` that aren't available in the monorepo (and so in verdaccio when running the e2e)
 
## How to test

- Check on CI

### CODE REVIEW

**Comment by gaetanmaisse** [comment]
Everything looks ok on CI BUT there are still some weird stuff around, here are the logs after an e2e run for `react`:

```shell
yarn build-storybook --quiet
info @storybook/react v6.3.0-alpha.42
info 
info => Cleaning outputDir: /tmp/storybook-e2e-testing/react/storybook-static
info => Loading presets
Error: Cannot find module 'typescript'
Require stack:
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/react-docgen-typescript-plugin/dist/index.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/react/dist/cjs/server/framework-preset-react-docgen.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/core-common/dist/cjs/presets.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/core-common/dist/cjs/index.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/core-server/dist/cjs/index.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/core/dist/cjs/server.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/core/server.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/react/dist/cjs/server/build.js
- /tmp/storybook-e2e-testing/react/node_modules/@storybook/react/bin/build.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.resolve (internal/modules/cjs/helpers.js:78:19)
    at Object.<anonymous> (/tmp/storybook-e2e-testing/react/node_modules/@storybook/react-docgen-typescript-plugin/dist/index.js:11:13)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/tmp/storybook-e2e-testing/react/node_modules/@storybook/react/dist/cjs/server/framework-preset-react-docgen.js:11:59)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/react-docgen-typescript-plugin/dist/index.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/react/dist/cjs/server/framework-preset-react-docgen.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/core-common/dist/cjs/presets.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/core-common/dist/cjs/index.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/core-server/dist/cjs/index.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/core/dist/cjs/server.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/core/server.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/react/dist/cjs/server/build.js',
    '/tmp/storybook-e2e-testing/react/node_modules/@storybook/react/bin/build.js'
  ]
}
info => Compiling manager..
info => Compiling preview..
info => Loading 1 config file in "/tmp/storybook-e2e-testing/react/.storybook"
info => Loading 7 other files in "/tmp/storybook-e2e-testing/react/.storybook"
info => Adding stories defined in "/tmp/storybook-e2e-testing/react/.storybook/main.js"
info => Using implicit CSS loaders
info => Using default Webpack4 setup
(node:867) DeprecationWarning: Default PostCSS plugins are deprecated. When switching to '@storybook/addon-postcss',
you will need to add your own plugins, such as 'postcss-flexbugs-fixes' and 'autoprefixer'.

See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-default-postcss-plugins for details.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
[BABEL] Note: The code generator has deoptimised the styling of /tmp/storybook-e2e-testing/react/node_modules/prettier/parser-flow.js as it exceeds the max of 500KB.
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
[BABEL] Note: The code generator has deoptimised the styling of /tmp/storybook-e2e-testing/react/node_modules/prettier/parser-typescript.js as it exceeds the max of 500KB.
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -220,6 +221,7 @@ jobs:
           name: Run E2E tests
           # Do not test CRA here because it's done in PnP part
           command: yarn test:e2e-framework vue3 angular angular11 web_components_typescript
+          no_output_timeout: 30m
```
Comment (gaetanmaisse) [inline .circleci/config.yml:224]: 🤦🏻   should be 5m...

**Review by shilman — APPROVED** [review]
💯

**Comment by shilman** [comment]
@gaetanmaisse this was fixed but hasn't been published yet:

```
Error: Cannot find module 'typescript'
```

**Comment by merceyz** [comment]
Shouldn't the test be failing due to https://app.circleci.com/pipelines/github/storybookjs/storybook/18720/workflows/5e5c7e33-5801-4f5a-bcc2-e752163d30ad/jobs/319998?invite=true#step-105-86 ?

**Comment by gaetanmaisse** [comment]
I was thinking that too, what's very strange is that Cypress tests are running and 🟢 , meaning the bundled app is basically working... I think I'm missing some knowledge about where/how `@storybook/react-docgen-typescript-plugin` is used

**Comment by shilman** [comment]
I published a new version and upgraded, so hopefully this goes away 🙏

**Pull request merged**